src/qml/pages/NewChatPage.qml

```diff
@@ -54,11 +54,8 @@ OperationPage {
         }
     }
 
-    Item {
+    SearchContactsList {
```

Comment:
SelectContactsList sounds better for me. Because selection is the central action here.
Moreover, let's add property "multiselect: false" later. For new group creation we will set it to true